bip-internalkey.md

```diff
@@ -0,0 +1,70 @@
+```
+BIP: XXX
+Layer: Consensus (soft fork)
+Title: OP_INTERNALKEY
+Author: Brandon Black <freedom@reardencode.com>, Jeremy Rubin <j@rubin.io>
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-XXXX
+Status: Draft
+Type: Standards Track
+Created: 2023-12-22
+License: BSD-3-Clause
+```
+
+## Abstract
+
+This BIP describes a new tapscript opcode (`OP_INTERNALKEY`) which
+pushes the taproot internal key to the stack.
+
+## Specification
+
+When verifying taproot script spends having leaf version `0xc0` (as defined in
+[BIP 342]), `OP_INTERNALKEY` replaces `OP_SUCCESS203` (0xcb). `OP_INTERNALKEY`
+pushes the taproot internal key, as defined in [BIP 341], to the stack.
```

Comment:
nit: might be nice to mark "taproot internal key" as a formal term, either by italicizing, backticks or the like. I find that marking terms that invoke a suite of definitions and properties is really helpful. We do it a lot in the BOLTs and I'd recommend doing it in the BIPs as much as we can.